src/spec.h

```diff
@@ -305,7 +304,7 @@ typedef struct IndexSpec {
   GCContext *gc;                  // Garbage collection
 
   SynonymMap *smap;               // List of synonym
-  char **aliases;                 // Aliases to self-remove when the index is deleted
+  HiddenString **aliases;         // Aliases to self-remove when the index is deleted
```

Comment:
(Not related to this line)
The stopwords should also use the HiddenString API right?